thewalrus/tests/test_quantum.py

```diff
@@ -55,6 +55,9 @@
     update_probabilities_with_noise,
     loss_mat,
     fidelity,
+    normal_ordered_expectation,
+    photon_number_expectation,
+    photon_number_squared_expectation,

```

Comment:
How about adding a test case for `gen_normal_ordered_complex_cov`?

Reply:
In principle any test for the other function in this PR is implicitly testing that  this function is doing what it is supposed to do.

Reply:
That's true! One thing to keep in mind, however, is that if a test case depends on this function and fails, then it can be hard to track down where the failing bit of code is. On the other hand, if `gen_normal_ordered_complex_cov` is unit tested, then we can be sure that the problem comes from another part of the codebase.